src/environments/environment.ts

```diff
@@ -45,21 +21,37 @@ const mockProviders: Provider[] = [
   },
 ];
 
+const pageConfig = {
```

Comment:
I find the pageConfig object could be confusing things, and make people think we want to keep the web-sdk module and the lean page around

Reply:
I removed the pageConfig completely